### PR TITLE
[RFR] - Classcontent controller rights

### PR DESCRIPTION
**Pull request by mickaelandrieu**
* Add rights on Controller
* Checks tests

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
@eric-chau ready for review \o/

File: Rest/Controller/ClassContentController.php [hunk 1]
```diff
@@ -458,9 +490,10 @@ private function sortDraftCollection(array $drafts)
...
      * Updates and returns content and its draft according to provided data.
      *
-     * @param  string $type
-     * @param  string $uid
-     * @param  array $data
+     * @param string $type
+     * @param string $uid
+     * @param array  $data
+     *
      * @return AbstractClassContent
```
Comment (eric-chau) [inline Rest/Controller/ClassContentController.php]: You should also add this =)

```php
/**
 * @Rest\Security("is_fully_authenticated() & has_role('ROLE_API_USER')")
 */
```

**Comment by eric-chau** [comment]
:+1:, thanks for your contribution @mickaelandrieu

**Comment by mickaelandrieu** [comment]
I rebase and then I merge :) thanks for review :)

**Pull request merged**